lib/broadside/cli/gli_commands.rb

```diff
@@ -17,6 +17,11 @@ def add_instance_flag(cmd)
   cmd.flag [:n, :instance], type: Fixnum
 end
 
+def add_command_flags(cmd)
```

Comment:
i think quite the opposite and the reason is because it prevents errors like leaving out the `add_tag_flag` when writing the code.  that's the whole point of doing stuff DRY - so you don't make a mistake when you are doing things over and over.